Content.Server/Fax/FaxSystem.cs

```diff
@@ -416,13 +416,20 @@ public void PrintFile(EntityUid uid, FaxMachineComponent component, FaxFileMessa
         component.SendTimeoutRemaining += component.SendTimeout;
 
         UpdateUserInterface(uid, component);
+
+        if (args.Session.AttachedEntity != null)
+            _adminLogger.Add(LogType.Action, LogImpact.Low,
+                $"{ToPrettyString(args.Session.AttachedEntity.Value):actor} added print job to {ToPrettyString(uid):tool} with text: {args.Content}");
+        else
+            _adminLogger.Add(LogType.Action, LogImpact.Low,
+                $"Someone added print job to {ToPrettyString(uid):tool} with text: {args.Content}");
```

Comment:
This is misleading... the `uid` is of the fax machine the player is interacting with, not the receiver of the print job.

Reply:
> Isn't the receiving fax machine for file prints always the one that the player is interacting with? The signature of functions like this one usually has the entity ID be the owner of the component that is being passed

Oh, you are absolutely right; I mixed up which method body I was looking at.